exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/main_config_ui.ipynb

```diff
@@ -113,8 +113,10 @@
     "                CKey.db_encryption)\n",
     "        ],\n",
     "        [\n",
-    "            ('Host Name', widgets.Text(value=conf.get(CKey.bfs_host_name, '127.0.0.1')), CKey.bfs_host_name),\n",
-    "            ('Port', widgets.IntText(value=int(conf.get(CKey.bfs_port, '2580'))), CKey.bfs_port),\n",
+    "            ('External Host Name', widgets.Text(value=conf.get(CKey.bfs_host_name, 'localhost')), CKey.bfs_host_name),\n",
+    "            ('Internal Host Name', widgets.Text(value=conf.get(CKey.bfs_host_name, 'localhost')), CKey.bfs_internal_host_name),\n",
+    "            ('External Port', widgets.IntText(value=int(conf.get(CKey.bfs_port, '2580'))), CKey.bfs_port),\n",
+    "            ('Internal Port', widgets.IntText(value=int(conf.get(CKey.bfs_port, '2580'))), CKey.bfs_internal_port),\n",
```

Comment:
```suggestion
    "            ('Internal Port', widgets.IntText(value=int(conf.get(CKey.bfs_internal_port, '2580'))), CKey.bfs_internal_port),\n",
```

Reply:
I wanted to preserve compatibility with existing installations. Changing the names as you suggested will invalidate the existing configuration.